synchronizer/l2_sync/l2_shared/batch_compare.go

```diff
@@ -75,3 +109,9 @@ func AreEqualStateBatchAndTrustedBatch(stateBatch *state.Batch, trustedBatch *ty
 	}
 	return false, debugStrResult
 }
+
+func hash(data []byte) common.Hash {
+	sha := sha3.NewLegacyKeccak256()
+	sha.Write(data)
+	return common.BytesToHash(sha.Sum(nil))
+}
```

Comment:
What about putting this function in the state helper file?

Reply:
I agree, good idea. I'll create another PR

Reply:
Done in current PR